Source/Scene/Model.js

```diff
@@ -612,10 +672,57 @@ define([
         }
     });
 
+    function getBasePath(url) {
+        var basePath = '';
+        var i = url.lastIndexOf('/');
+        if (i !== -1) {
+            basePath = url.substring(0, i + 1);
+        }
+
+        return basePath;
+    }
+
+    function getAbsoluteURL(url) {
+        var docUri = new Uri(document.location.href);
+        var modelUri = new Uri(url);
+        return modelUri.resolve(docUri).toString();
+    }
+
+    var sizeOfUnit32 = Uint32Array.BYTES_PER_ELEMENT;
+
+    function parseBinaryGltfHeader(arrayBuffer) {
+        var magic = getStringFromTypedArray(arrayBuffer, 0, 4);
+        if (magic !== 'glTF') {
+            throw new DeveloperError('bgltf is not a valid Binary glTF file.');
+        }
+
+        var view = new DataView(arrayBuffer);
+        var byteOffset = 0;
+
+        byteOffset += sizeOfUnit32;  // Skip magic number
+
+        var version = view.getUint32(byteOffset, true);
+        byteOffset += sizeOfUnit32;
+        if (version !== 1) {
+            throw new DeveloperError('Only glTF Binary version 1 is supported.  Version ' + version + ' is not.');
```

Comment:
This looks more like a `RuntimeError` to me.  If you disagree, it needs to be wrapped in a pragma